aries-site/src/pages/components/toast-notifications.mdx

```diff
@@ -0,0 +1,100 @@
+
+### Status Indicator
+
+Avoid using toasts for very critical information or alerts. Users may accidentally dismiss or entirely miss a toast, so it is best to avoid this case for important information. Additionally, avoid using toasts, critical or not, if it requires a user action. 
+
+The status indicators used for toasts are strictly for OK, Warning, and Unknown statuses.
+
+<Box direction="row" gap="medium" pad={{bottom: 'medium'}}>
+    <StatusGoodSmall color="status-ok" />
+    <StatusWarningSmall color="status-warning" />
+    <StatusUnknownSmall color="status-unknown" />
+</Box>
+
+You can read more guidance about status indicators [here on this page](/components/status-indicator).
+
+### Content (Title + Message)
+
+All toasts have content which is comprised of a title and a message. However, the content only requires a title, not necessarily a message. The title and message should be distinguishable from each other by the weight of the text. The status title and message should be distinguishable by the weight of the text.
+
+1. Amount of Information
+
+Toast notifications should not have too much information inside of them. Additionally, each notification should only revolve around one idea or subject. Because a toast notification is only present on the screen for a short amount of time, a user using assistive technology needs to have an understanding of what information is in the notification. Therefore, the content should be minimal and direct.
+
+For best practice, keep the amount of information to three lines or under. This three line limit either can be a combination of both the title and the message or a three-lined title. If the information is heavy and long, consider using a different type of notification other than toast. NEVER truncate text within a toast. The main point is to keep the information concise and clear.
+
+2. Interactive Content
+
+Avoid placing interactive elements as this violates WCAG 2.1 guidelines, like links and buttons, inside of a toast. These elements may be difficult to access with a screen reader and other assistive technology, so it is best to avoid complex toasts.
+
+### Close Button
+
+The close button within a toast should be the only interactable element and is used to dismiss the notification. To clarify, the toast itself is not dismissable on click rather only when the close button is specifically activated. This is to avoid any confusion the user may have on what the interactable elements are within a toast.
+
+## Placement 
+
+Make sure to follow these guidelines for Toast Notifications:
+- Place them either top-center or top-right of the screen. If badging is being used, toasts should be placed underneath it. Note that top-right placements may negatively affect those using screen magnifiers.
+- Fade in on arrival and fade out when dismissed.
+- Do not obscure critical information and any primary action elements.
```

Comment:
Agreed with this point. However, often times, page level actions live at the top right of our layouts. Will this conflict? Or, where should toasts go and how do we know what kind of layout a user is going to be on when a toast appears?